AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -414,48 +434,129 @@ public async Task<ActionResult> GetHearingById(Guid hearingId)
+        [ProducesResponseType((int) HttpStatusCode.NotFound)]
+        [ProducesResponseType((int) HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> GetHearingConferenceStatus(Guid hearingId)
+        {
+            var errorMessage =
+                $"Failed to get the conference from video api, possibly the conference was not created or the kinly meeting room is null - hearingId: {hearingId}";
+            try
+            {
+                _logger.LogDebug($"Hearing {hearingId} is confirmed. Polling for Conference in VideoApi");
+                var conferenceDetailsResponse =
+                    await _conferenceDetailsService.GetConferenceDetailsByHearingId(hearingId);
+                _logger.LogInformation($"Found conference for hearing {hearingId}");
+                if (conferenceDetailsResponse.HasValidMeetingRoom())
+                {
+                    return Ok(new UpdateBookingStatusResponse
+                    {
+                        Success = true,
+                        TelephoneConferenceId = conferenceDetailsResponse.MeetingRoom.TelephoneConferenceId
+                    });
+                }
+                else
+                {
+                    return Ok(new UpdateBookingStatusResponse {Success = false, Message = errorMessage});
+                }
+            }
+            catch (VideoApiException ex)
+            {
+                _logger.LogError(ex, "Failed to confirm a hearing. {ErrorMessage}", errorMessage);
+                _logger.LogError("There was an unknown error for hearing {Hearing}. Updating status to failed",
+                    hearingId);
+
+                // Set the booking status to failed as the video api failed
+                //await UpdateFailedBookingStatus(hearingId);
+
+                return Ok(new UpdateBookingStatusResponse {Success = false, Message = errorMessage});
+            }
+        }
+
+        /// <summary>
+        ///     Update the failed hearing status.
+        /// </summary>
+        /// <param name="hearingId">The hearing id</param>
+        /// <returns>Success status</returns>
+        [HttpGet("{hearingId}/update-status")]
+        [SwaggerOperation(OperationId = "UpdateFailedBookingStatus")]
+        [ProducesResponseType(typeof(UpdateBookingStatusResponse), (int) HttpStatusCode.OK)]
+        [ProducesResponseType((int) HttpStatusCode.NotFound)]
+        [ProducesResponseType((int) HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> UpdateHearingStatus(Guid hearingId)
+        {
+            var errorMessage =
+                $"Failed to get the conference from video api, possibly the conference was not created or the kinly meeting room is null - hearingId: {hearingId}";
+            try
+            {
+                await UpdateFailedBookingStatus(hearingId);
+                return Ok(new UpdateBookingStatusResponse {Success = false, Message = errorMessage});
+            }
+            catch (Exception e)
```

Comment:
what happens if the hearing doesn't exist? Could we add a test to assert that it returns a 404?